homeassistant/components/notify/pushbullet.py

```diff
@@ -150,18 +152,20 @@ def _push_data(self, message, title, data, pusher, tname=None):
                     if filedata.get('file_type') == 'application/x-empty':
                         _LOGGER.error("Can not send an empty file")
                         return
-                    pusher.push_file(title=title, body=message, **filedata)
+
+                    pusher.push_file(title=title, body=message,
+                                     **email_kwargs, **filedata)
```

Comment:
This syntax is only allowed in Python 3.5 and later ([pep](https://www.python.org/dev/peps/pep-0448/)). Python 3.4 support has not been dropped yet.

Reply:
Bummer, unfortunately the pushbullet REST api chokes on the email parameter being set on any target not email even if it is set to none. I'll poke some more to see if I can work around it but it may just end up having to be all the if/else statements unless there is another suggestion.

Reply:
Ok, I must be special, retested and it works fine setting the email parameter to None. It breaks if testing manually but the pushbullet python library handles it fine.

Reply:
Just merge the 2 dicts and unpack the merged one.